### PR TITLE
feat(design): Card Component

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
`daff-card` molecule component.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/design/src/molecules/card/card/card.component.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import '../../../scss/daff-util';
+
+.daff-card {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  max-width: 600px;
```
Comment (damienwebdev) [inline libs/design/src/molecules/card/card/card.component.scss]: I don't think we should be enforcing a max-width.

File: apps/design-land/src/app/card/card.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>Daff Card</h1>
+
+<h2>Card</h2>
+<daff-card>
+  <img daffCardImage src="/assets/card/uber-logo.svg" />
+  <h4 daffCardTitle>Title</h4>
+  <p daffCardBody>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```
Comment (damienwebdev) [inline apps/design-land/src/app/card/card.component.html]: I don't like this API. I don't want to have to add `daffCardBody` to every element I want inside a `card`.

Ideally, I think that `daffCardBody` should be `implicit` with an API like:

```
<daff-card>
  <img daffCardImage src="/assets/card/uber-logo.svg" />
  <h4 daffCardTitle>Title</h4>
  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
</daff-card>
```

makes more sense.

File: libs/design/src/molecules/card/card/card.component.scss (hunk 1 above)
Comment (xelaint) [reply libs/design/src/molecules/card/card/card.component.scss]: Resolved in 22430c3

File: apps/design-land/src/app/card/card.component.html (hunk 2 above)
Comment (xelaint) [reply apps/design-land/src/app/card/card.component.html]: Resolved in 22430c3

File: libs/design/src/molecules/card/card/card.component.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Component, ViewEncapsulation, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+  selector: 'daff-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss'],
+  host: {
+    'class': 'daff-card'
```
Comment (damienwebdev) [inline libs/design/src/molecules/card/card/card.component.ts]: Needs test coverage.

Comment (xelaint) [reply libs/design/src/molecules/card/card/card.component.ts]: Resolved in d3acb9e

**Comment by xelaint** [comment]
@damienwebdev squashed

**Pull request merged**